### PR TITLE
Create 6.0.0.md

### PR DESCRIPTION
**Pull request by wilr**
## Issue
- https://github.com/silverstripe/silverstripe-framework/issues/11167

### CODE REVIEW

File: en/08_Changelogs/6.0.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+# 6.0.0 (unreleased)
+
+## Overview
+
+- [Run CanonicalURLMiddleware in all environment by default](#url-middleware)
+- [Bug fixes](#bug-fixes)
+
+## Run CanonicalURLMiddleware in all environment by default (#url-middleware)
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
## Run `CanonicalURLMiddleware` in all environments by default {#url-middleware}
```

File: en/08_Changelogs/6.0.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: 6.0.0 (unreleased)
+---
+
+# 6.0.0 (unreleased)
+
+## Overview
+
+- [Run CanonicalURLMiddleware in all environment by default](#url-middleware)
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
- [Run `CanonicalURLMiddleware` in all environments by default](#url-middleware)
```

File: en/08_Changelogs/6.0.0.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- [Bug fixes](#bug-fixes)
+
+## Run CanonicalURLMiddleware in all environment by default (#url-middleware)
+
+In Silvestripe 5.0 `CanonicalURLMiddleware` only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in dev, test and live by default.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
In Silverstripe CMS 5 the [`CanonicalURLMiddleware`](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in `dev`, `test`, and `live` by default.
```
Please also include a short yaml snippet showing how to change this (or link to existing docs which show it)

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Thanks!
Just a few small changes (plus whatever the CI is complaining about - might just be the heading change that I've already requested)

File: en/08_Changelogs/6.0.0.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+To opt out of this change include the following in your `_config.php`
+
+```php
+use SilverStripe\Core\CoreKernel;
+
+if (Director::isDev()) {
+    CanonicalURLMiddleware::singleton()->setEnabledEnvs([
+        CoreKernel::LIVE,
+    ]);
+}
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
CanonicalURLMiddleware::singleton()->setEnabledEnvs([
    CoreKernel::LIVE,
]);
```
Saying "set it to live when I'm in dev mode" is confusing at best.

Comment (ChloeSartorelli) [reply en/08_Changelogs/6.0.0.md]: Please also add a `use` statement for `CoreKernel`.

File: en/08_Changelogs/6.0.0.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+# 6.0.0 (unreleased)
+
+## Overview
+
+- [Run `CanonicalURLMiddleware` in all environments by default](#url-middleware)
+- [Bug fixes](#bug-fixes)
+
+## Run `CanonicalURLMiddleware` in all environments by default {#url-middleware}
+
+In Silverstripe CMS 5 [`CanonicalURLMiddleware`](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in dev, test and live by default.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
In Silverstripe CMS 5 [`CanonicalURLMiddleware`](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in `dev`, `test`, and `live` by default.
```
Originally requested in https://github.com/silverstripe/developer-docs/pull/465#discussion_r1510523381

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/6.0.0.md (hunk 4 above)
Comment (wilr) [reply en/08_Changelogs/6.0.0.md]: @GuySartorelli That's the craziness of the current behaviour!

Comment (ChloeSartorelli) [reply en/08_Changelogs/6.0.0.md]: Oh! Sorry I thought it was the `CoreKernel` use statement that was missing, but it's `CanonicalURLMiddleware`'s use statement that's missing. Can you please add that?

**Review by GuySartorelli — APPROVED** [review]
Great, thanks!

**Pull request merged**